pynfe/utils/__init__.py

```diff
@@ -18,9 +18,14 @@
 
 
 # @memoize
-def so_numeros(texto):
-    """Retorna o texto informado mas somente os numeros"""
-    return "".join(filter(str.isdigit, str(texto)))
+def so_numeros(texto) -> str:
```

Comment:
```suggestion
def so_numeros(texto: str) -> str:
```

Reply:
More clearly what are the inputs and outputs.